CHANGELOG.md

```diff
@@ -4,13 +4,14 @@ We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
 ## Unreleased
 
 ### Added
-- 
+-
 
 ### Changed
-- 
+- **cf-notifications:** [PATCH] Renamed from `cf-notification`
+to `cf-notifications`.
```

Comment:
Do we enforce 80 character line lengths in markdown files?

Reply:
I usually wrap them. The only practical reason to enforce them at all is to prevent scrolling in the GH UI, right, which applies to all text files?

Reply:
I thought it had to do with diffing and that markdown files should be have newlines at the end of discrete thoughts.

Reply:
Should I revert it? I've been wrapping lines in cfgov-refresh changelog for ages 😱

Reply:
As you both noted, there are two aspects to this:
1. Breaking lines at discrete thoughts can make diffing future changes easier.
2. Some max length should be adhered to so that lines aren't horizontally scrolling in the GH UI.

The two are mostly unrelated. The max length could be longer than 80 characters, but I don't mind using that as our guideline, since that's what we enforce for CSS, JS, and Python.

In this case, I would ask that successive lines of bullets be indented to match the first non-bullet-non-whitespace character of the first line, though. Makes reading lists in raw Markdown easier.

Reply:
@anselmbradford It doesn't matter to me. Merge at will.